go.sum

```diff
@@ -0,0 +1,53 @@
+cloud.google.com/go v0.34.0 h1:eOI3/cP2VTU6uZLDYAoic+eyzzB9YyGmJ7eIjl8rOPg=
```

Comment:
What's the rationale for checking this in? Genuinely not familiar with how these are intended to work.

Reply:
go.sum is checksums of dependencies. It's used to verify whether the packages you've downloaded are in fact the packages that you intended to fetch. Per `go help modules` output:

> The go command maintains, in the main module's root directory alongside
> go.mod, a file named go.sum containing the expected cryptographic checksums
> of the content of specific module versions. Each time a dependency is
> used, its checksum is added to go.sum if missing or else required to match
> the existing entry in go.sum.